apiserver/params/crossmodel.go

```diff
@@ -367,6 +367,10 @@ type RemoteRelationChangeEvent struct {
 
 	SuspendedReason string `json:"suspended-reason,omitempty"`
 
+	// Settings represent the updated application-level settings in
+	// this relation.
+	Settings map[string]interface{} `json:"settings,omitempty"`
```

Comment:
Can we call this ApplicationSettings so that when it's used in code elsewhere it's not just foo.Settings = blah but foo.ApplicationSettings = blah and it's obvious to the reader what settings it is (could be confused with unit settings etc).

Reply:
Yup